src/tools/haxelib/Main.hx

```diff
@@ -252,7 +254,7 @@ class Main {
 			if (cats[i] == null) cats[i] = [c];
 			else cats[i].push(c);
 		}
-		
+
```

Comment:
`static public function ask( question )`
Ok, I'll move it to something like Cli or CliUtil.
`print`& `command` too.